graphql-anycable.gemspec

```diff
@@ -30,11 +30,13 @@ Gem::Specification.new do |spec|
   spec.add_dependency "anyway_config", ">= 1.3", "< 3"
   spec.add_dependency "graphql",       ">= 1.11", "< 3"
   spec.add_dependency "redis",         ">= 4.2.0"
+  spec.add_dependency "activejob",     ">= 5.2.0"
```

Comment:
It shouldn't be a dependency; the gem is Rails-free, we only provide ActiveJob integration if ActiveJob is available in the runtime.

Reply:
I added it to `spec.add_development_dependency "activejob", ">= 6.0.0"`

because otherwise, I got an error during running specs